go-controller/pkg/ovn/gress_policy.go

```diff
@@ -95,11 +95,17 @@ func (gp *gressPolicy) ensurePeerAddressSet(factory AddressSetFactory) error {
 	return nil
 }
 
-func (gp *gressPolicy) addPeerPod(pod *v1.Pod) error {
+func (gp *gressPolicy) addPeerPod(pod *v1.Pod, factory AddressSetFactory) error {
 	ips, err := util.GetAllPodIPs(pod)
 	if err != nil {
 		return err
 	}
+
+	if err := gp.ensurePeerAddressSet(factory); err != nil {
```

Comment:
I don't think this is necessary; if the initial `ensurePeerAddressSet` fails, it won't continue setting up the `gressPolicy`, so this will never get called. (Likewise in `deletePeerPod`)

Reply:
I see because wont add the handler to the policyHandlers, OK will remove it